client/src/components/SearchablePooledMultiSelectDropdown.js

```diff
@@ -0,0 +1,170 @@
+import React, { Component } from 'react';
```

Comment:
This COMPONENT is currently unused and will become an alternative to the `PooledMultiSelectDropdown` once bugfix is published by semantic ui